cypress/support/pages/yves/cart/cart-page.ts

```diff
@@ -11,6 +11,12 @@ export class CartPage extends YvesPage {
 
   protected PAGE_URL = '/cart';
 
+  visitCartWithItems = (): void => {
+    this.visit();
+    this.repository.getCartUpsellingAjaxLoader().should('be.visible');
+    this.repository.getCartUpsellingAjaxLoader().should('be.not.visible');
```

Comment:
Please try to avoid assertions in page objects, plus it's not important for us. It looks like component tests, not E2E.